src/main/org/deidentifier/arx/ARXLattice.java

```diff
@@ -1265,6 +1263,13 @@ private void readObject(ObjectInputStream aInputStream) throws ClassNotFoundExce
         }
     }
     
+    private void setMonotonicity(boolean isSuppressionAlwaysEnabled, int absoluteMaxOutliers) {
```

Comment:
When testing ARXConfiguration changes, I knew what to look for and have general confidence in the effect of the changes.  I'm not as confident in the ARXLattice changes.  Seems like these evaluations drive a specific optimization of the search process, replacing utility IL bounds with node IL bounds.  I tested as no change from baseline but haven't configured a test of the "new code path" that would be used by utilities with non-monotonic generalization.